src/components v2/Atoms/Forms/InputSelect/InputSelect.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="ph-input-autocomplete-select">
+    <P2Autocomplete
+      :show-footer="showFooter"
+      :custom-label-var="customLabelVar"
+      :open-indicator-icon="openIndicatorIcon"
+      :option-items="items"
+      :init-input="value"
+      :value="value"
+      :disabled="disabled"
+      hide-clear-btn
+      @input="$emit('input', selected)"
+      @update:selected="selected = $event"
+    />
```

Comment:
**Question:** the input select should have no type-ahead field. Are these props making sure that that is the case?